docs/quickstart.md

```diff
@@ -1,246 +1,67 @@
-        "createdOn": 1572379783,
-        "id": "123456789012",
-        "lastModifiedOn": 1572381585,
-        "metadata": null,
-        "principalPolicyHash": "\"852ee9abbf1220a111c435a8c0e65490\"",
-        "principalRoleArn": "arn:aws:iam::123456789012:role/DCEPrincipal"
-    }
-]
-```
-Once you see the first lease provisioned in the system, you are ready to use your
-first lease! See [logging into your leased account](howto.md#login-to-your-dce-account).
-
-### Listing leases
-
-To list the leases, use an HTTP GET request to the *${api_url}/leases* endpoint
-to see the response as shown here:
-
-```json
-[
-    {
-        "accountId": "123456789012",
-        "budgetAmount": 20,
-        "budgetCurrency": "USD",
-        "budgetNotificationEmails": [
-            "myuser@example.com"
-        ],
-        "createdOn": 1572381585,
-        "expiresOn": 1572382800,
-        "id": "94503268-426b-4892-9b53-3c73ab38aeff",
-        "lastModifiedOn": 1572381585,
-        "leaseStatus": "Active",
-        "leaseStatusModifiedOn": 1572381585,
-        "leaseStatusReason": "Active",
-        "principalId": "DCEPrincipal"
-    }
-]
-```
-
-### Destroying leases
-
-Leases can automatically expire based on a date or a budget amount, but
-leases may also be administratively destroyed at any time. To destroy
-a lease with the API, send a DELETE request to *${api_url}/leases
-with the following request body:
-
-```json
-{
-    "principalId": "DCEPrincipal",
-    "accountId": "123456789012"
-}
-```
-
-The API response for a successful lease destroy looks like this:
-
-```json
-{
-    "accountId": "519777115644",
-    "budgetAmount": 20,
-    "budgetCurrency": "USD",
-    "budgetNotificationEmails": [
-        "john.doe@example.com"
-    ],
-    "createdOn": 1572381585,
-    "expiresOn": 1572382800,
-    "id": "94503268-426b-4892-9b53-3c73ab38aeff",
-    "lastModifiedOn": 1572442028,
-    "leaseStatus": "Inactive",
-    "leaseStatusModifiedOn": 1572442028,
-    "leaseStatusReason": "Destroyed",
-    "principalId": "jdoe123"
-}
-```
+Deploy DCE and lease an account quickly using the DCE CLI.
+
+1. Download the appropriate executable for your OS from the [latest release](https://github.com/Optum/dce-cli/releases/latest). e.g. for mac, you should download dce_darwin_amd64.zip
+
+1. Unzip and move the executable to a directory on your PATH, e.g.
+
+    ```
+    # Download the zip file
+    wget https://github.com/Optum/dce-cli/releases/download/<VERSION>/dce_darwin_amd64.zip
+
+    # Unzip to a directory on your path
+    unzip dce_darwin_amd64.zip -d /usr/local/bin
+    ```
+
+1. Type `dce init`. Leave all fields blank for now.
+
+1. Deploy DCE using IAM Credentials that have AdministratorAccess
+    ```
+    export AWS_ACCESS_KEY_ID=XXXXXXXXXX
+    export AWS_SECRET_ACCESS_KEY=XXXXXXXXXXXXXXXXXXXX
+    dce system deploy
+    ```
+
+1. Retrieve the DCE API url from API Gateway in your master account, and add it to the dce config file, e.g.
```

Comment:
> Retrieve the DCE API url from API Gateway in your master account

How does the user find this?
I think `dce system deploy` will spit this out, no? or is that hard for the user to find?

Reply:
We could go a step further and populate .dce/config with the TF outputs on deploy complete.

Reply:
It's currently too hard to find. I think this is something we want to automate in the near future as well.